src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -115,30 +115,44 @@
       @end
       function {@class.initClientFuncName}() {
         gapi.client.init({
-          'apiKey': '',  // TODO: Update placeholder with desired API key.
-          @if class.auth.type != "API_KEY"
+          @if class.auth.type != "NONE"
+            'apiKey': '',  // TODO: Update placeholder with desired API key.
```

Comment:
In the JS generator review Bochun stated that the API key should be included for all auth types, so I believe so